crates/air_r_formatter/src/r/auxiliary/call_arguments.rs

```diff
@@ -76,6 +76,326 @@ impl RCallLikeArguments {
             Self::Subset2(node) => node.syntax(),
         }
     }
+
```

Comment:
This is now a method on `RCallLikeArguments`, hence the big diff as the whole thing moved. It feels right and this gives access to `self` and the syntax elements.